pkg/expr/sql/parser.go

```diff
@@ -1,200 +1,72 @@
 package sql
 
 import (
-	"errors"
+	"encoding/json"
+	"fmt"
+	"sort"
 	"strings"
 
-	parser "github.com/krasun/gosqlparser"
-	"github.com/xwb1989/sqlparser"
+	"github.com/jeremywohl/flatten"
+	"github.com/scottlepp/go-duck/duck"
+)
+
+const (
+	TABLE_NAME    = "table_name"
+	ERROR         = ".error"
+	ERROR_MESSAGE = ".error_message"
 )
 
 // TablesList returns a list of tables for the sql statement
-// TODO: should we just return all query refs instead of trying to parse them from the sql?
 func TablesList(rawSQL string) ([]string, error) {
-	stmt, err := sqlparser.Parse(rawSQL)
+	duckDB := duck.NewInMemoryDB()
+	cmd := fmt.Sprintf("SELECT json_serialize_sql('%s')", rawSQL)
```

Comment:
I bet there are some escaping issues here.   Try a query with quotes like
```
"SELECT json_serialize_sql('SELECT json_serialize_sql('SELECT 1')')
```

Reply:
fixed.  added a couple test cases.  but they are skipped for now.  need to add duckdb cli to the ci image